tools/depends/target/Toolchain_binaddons.cmake.in

```diff
@@ -4,6 +4,9 @@ set(CPU "@use_cpu@")
 set(PLATFORM "@target_platform@")
 set(APP_RENDER_SYSTEM @app_rendersystem@)
 
+# FIXME: Addons need to set this individually, and this default needs to be removed.
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
For here not the best place as it is used only by Kodi's special addon build way (like jenkins use with your last call), if build is started independent is it not set.

e.g. my typical call:
```
mkdir build && cd build && cmake -DADDONS_TO_BUILD=$(basename $(dirname $(pwd))) -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=$HOME/.kodi/addons -DPACKAGE_ZIP=1  ~/Development/Kodi/kodi-Matrix/cmake/addons && make
```

Reply:
as the comment says: this is a default and should be overridden by each addon's CMakeLists.txt

Reply:
Where should this be set to also allow independent builds?

If everyone add-on must set this in their own CMakeLists.txt I'm not sure I like it. An add-on should be over to override this if it chooses to do so but a default is important.

Reply:
I agree, the default should stay

Reply:
Also not like to make on every addon itself as there are over 100 and on newer OS versions it can differ to older.
So we have not direct a overview where works correct and where not (by users and also by test systems).

For now we see exactly where is in miscompare to C++17, also if currently really a bad moment about.

I look currently to make a independent CompilerSettings.cmake for addons to use C++14 at the moment, but the best is to fix the addons about.

Reply:
> as the comment says: this is a default and should be overridden by each addon's CMakeLists.txt

About override is for addons CMakeLists.txt only possible for himself, but he have no affect and access to the related depends folder, to set there something else.